CHANGELOG.md

```diff
@@ -1,3 +1,31 @@
+## 2.33.0-rc.0 / 2022-01-12
+
+* [CHANGE] PromQL: Promote negative offset and `@` modifer to stable features. #10121
+* [CHANGE] Web: Promote remote-write-receiver to stable. #10119
+* [FEATURE] Config: Add `stripPort` template function. #10002
+* [FEATURE] Promtool: Add cardinality analysis to `check metrics`, enabled by flag `--extended`. #10045
+* [FEATURE] SD: Enable target discovery in own K8s namespace. #9881
+* [FEATURE] SD: Add provider ID label in K8s SD. #9603
+* [FEATURE] Web: Add limit field to the rules API. #10152
+* [ENHANCEMENT] Remote-write: Avoid allocations by buffering concrete structs instead of interfaces. #9934
+* [ENHANCEMENT] Remote-write: Log time series details for out-of-order samples in remote write receiver. #9894
+* [ENHANCEMENT] Remote-write: Shard up more when backlogged. #9274
+* [ENHANCEMENT] TSDB: Use simpler map key to improve exemplar ingest performance. #10111
+* [ENHANCEMENT] TSDB: Avoid allocations when popping from the intersected postings heap. #10092
+* [ENHANCEMENT] TSDB: Make chunk writing non-blocking, avoiding latency spikes in remote-write. #10051
+* [ENHANCEMENT] TSDB: Improve label matching performance. #9907
+* [ENHANCEMENT] UI: Rework service discovery page. #10131
+* [ENHANCEMENT] UI: Rework target page. #10103
```

Comment:
Would "rework" be the best word to define the user-facing changes? From the description of #10103, it seems the only changes that fall into that category on both pages are adding a search bar and infinite scroll, as well as a bugfix (https://github.com/prometheus/prometheus/issues/5219) specific to the targets page. I think the rework is mostly for the code.

cc @Nexucis

Reply:
I got the impression that the changes were quite visible, but I might have gotten the wrong impression here. Didn't look in detail. Happy to take suggestions. (Perhaps just "Improve" as a more generic word?)

Reply:
Indeed the result is quite visible @beorn7. But probably @LeviHarrison is right as well in a sense where maybe the word `rework` is a bit too strong from a user perspective.

Perhaps `[Enhancement] UI: Optimization of the target page (fix #6881) and provide a search bar` ?

Reply:
note that it's the same thing for the "service discovery"  page (i.e. optimization and a new search bar).

Reply:
That sounds more clear to me.

Reply:
Augustin's proposal SGTM but I would not mention the fixed issue number.